tests/ssg_test_suite/rule.py

```diff
@@ -68,24 +66,24 @@ def _send_scripts(domain_ip):
     logging.debug("Uploading scripts.")
     log_file_name = os.path.join(LogHelper.LOG_DIR, "data.upload.log")
 
-    command = "ssh {0} mkdir -p {1}".format(machine, remote_dir)
+    command = ("ssh", machine, "mkdir", "-p", remote_dir)
```

Comment:
Could the `run_cmd_remote` be used here instead?

Reply:
Unfortunatelly, `run_cmd_remote` has hardwired logging configuration, so it is not a drop-in replacement for this construct.